### PR TITLE
Update preset-env dependencies and fix fixtures

### PR DESCRIPTION
**Pull request by danez**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | n
| Patch: Bug Fix?          | y
| Major: Breaking Change?  | n
| Minor: New Feature?      | n
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

### CODE REVIEW

File: packages/babel-preset-env/test/debug-fixtures/entry-corejs3-versions-strings-minor-3.1/stdout.txt [hunk 1]
```diff
@@ -161,6 +162,7 @@ Using polyfills with `entry` option:
   es.string.includes { "ie":"10" }
   es.string.iterator { "ie":"10" }
   es.string.match { "ie":"10" }
+  es.string.match-all { "chrome":"54", "ie":"10", "node":"6.10" }
   es.string.pad-end { "chrome":"54", "ie":"10", "node":"6.10" }
   es.string.pad-start { "chrome":"54", "ie":"10", "node":"6.10" }
```
Comment (danez) [inline packages/babel-preset-env/test/debug-fixtures/entry-corejs3-versions-strings-minor-3.1/stdout.txt]: This is the only fixture that actually changes something. All the other ones just moved the import from a to b.

**Comment by zloirock** [comment]
Seems it would be better also update `core-js-compat` dependency.

**Comment by danez** [comment]
> Seems it would be better also update `core-js-compat` dependency.

It is using the latest version of `core-js-compat`

**Comment by zloirock** [comment]
I mean bump `preset-env` `core-js-compat` dependency in `package.json` to `^3.1.1`.

**Pull request merged**